simba_ml/prediction/time_series/config/transfer_learning_pipeline/data_config.py

```diff
@@ -14,3 +14,4 @@ class DataConfig:
     time_series: time_series_config.TimeSeriesConfig
     test_split: float = 0.2
     split_axis: str = "vertical"
+    export_path: str = None
```

Comment:
A string should not be None. You could make it `typing.Optional[str]`

Reply:
Okay, fixed it.